### ISSUE TITLE
Deploy the applications

### ISSUE THREAD

**Issue by MarcSkovMadsen** [ahuang11/panel-chat-examples#9]
It would be very cool if we got the applications deployed. For example to Hugging Face spaces.

As we don't have resources to cover api keys we would probably need to ask users for their api keys.

Instead of creating our own code and copying it into each app Panel could provide the functionality. I've made a feature request here https://github.com/holoviz/panel/issues/5562.

**Comment by ahuang11** [comment]
I think the first step is set max_token on each example. Once that's done, I think I can donate a set amount for user testing.

**Comment by ahuang11** [comment]
Actually maybe we can simply have a Mistral model demo; huggingface should support that right?

**Comment by MarcSkovMadsen** [comment]
Yes. There is also https://localai.io/basics/getting_started/index.html#run-localai-in-kubernetes

**Comment by MarcSkovMadsen** [comment]
One way to display some applications would be to use some simple to import libraries that works in pyodide like

- Talk about jokes https://github.com/pyjokes/pyjokes/blob/master/pyjokes/jokes_en.py
- Translate to pirate speech https://arrr.readthedocs.io/en/latest/
- transformers_js_py https://pypi.org/project/transformers-js-py/
